.github/workflows/cifuzz.yaml

```diff
@@ -13,12 +13,16 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18.x
     - name: Restore Go cache
       uses: actions/cache@v3
       with:
-        path: /home/runner/work/_temp/_github_home/go/pkg/mod
+        path: ~/go/pkg/mod
```

Comment:
I think for clarity it would still be a good idea to use the absolute path explicitly or rely on `GOMODCACHE` which is used in the makefile target to mount the volume.